### PR TITLE
Gui dav mount storage limit tests

### PR DESCRIPTION
**Pull request by maryvictol**
This PR adds tests for DAV Mount storage limit #2233

### CODE REVIEW

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+
+    @BeforeClass
+    public void createStorages() {
+        Stream.of(storage1, storage2, storage3)
+                .forEach(stor -> library()
+                    .createStorage(stor));
+    }
+
+    @BeforeMethod
+    void openApplication() {
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java]: I think it should be renamed to something like reLogin()?

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+                .selectStorage(storage1)
+                .showMetadata()
+                .ensure(FILE_SYSTEM_ACCESS, enabled)
+                .click(FILE_SYSTEM_ACCESS)
+                .ensure(FILE_SYSTEM_ACCESS, text(accessEnabledMessage()))
+                .ensure(DISABLE, enabled)
+                .sleep(10, SECONDS)
+                .inAnotherTab(webdavpage ->
+                        checkWebDavPage(() -> new ToolPageAO(endpoint)
+                                .assertPageTitleIs(format("Index of /webdav/%s", user.login.toUpperCase()))
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java]: "Index of /webdav/%s" can be introduced a constant

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+                    .selectByName(user.login)
+                    .showPermissions()
+                    .set(READ, ALLOW)
+                    .set(WRITE, ALLOW)
+                    .closeAll();
+            library()
+                    .selectStorage(storage1)
+                    .showMetadata()
+                    .click(FILE_SYSTEM_ACCESS)
+                    .ensure(FILE_SYSTEM_ACCESS, text("File system access enabled till"))
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java]: "File system access enabled till" can be introduced a constant

**Review by kamyshova — CHANGES_REQUESTED** [review]

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java (hunk 1 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java (hunk 2 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java (hunk 3 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java]: Done

**Review by kamyshova — APPROVED** [review]

**Pull request merged**